meteor/sofie-server/presentation/index.ts

```diff
@@ -0,0 +1,68 @@
+import cors from 'cors'
+import express, { Express, Router } from 'express'
+import { BaseController } from './controllers/base-controller'
+import { ControllerFacade } from './facades/controller-facade'
+import { RundownEventServerFacade } from './facades/rundown-event-server-facade'
+
+export * from './controllers/rundown-controller'
+
+const REST_API_PORT: number = 3005
+const RUNDOWN_EVENT_SERVER_PORT: number = 3006
```

Comment:
Should these possibly be moved to environment variables?

Reply:
Ideally, yes